Vagrantfile

```diff
@@ -6,8 +6,8 @@ Vagrant.configure("2") do |config|
     config.vm.box = "minimal/trusty64"
 
     # Create forwarding ports for client-guest machine access via localhost.
-    config.vm.network :forwarded_port, guest: 80, host: 8080
-    config.vm.network :forwarded_port, guest: 6000, host: 6000
+    config.vm.network :forwarded_port, guest: 80, host: 8080, auto_correct: true
+    config.vm.network :forwarded_port, guest: 5000, host: 5000, auto_correct: true
```

Comment:
The reason for 5000 is that Heroku uses that as a standard port. So if we auto correct, then there is no way we can keep 5000 for all vm in order to integrate with heroku. Maybe we can forward the same port to a different local port. So we keep the first 5000 and then forward to the host under a different port like 7777.